site/docs/master/namespace.md

```diff
@@ -22,4 +22,4 @@ velero client config set namespace=<NAMESPACE_VALUE>
 
 
 
-[0]: install-overview.md 
+[0]: advanced-installation.md 
```

Comment:
I think we want this to point to the "install the CLI" section in the getting-started doc

Reply:
Yeah, I think so too!